[2025-02-12T22:31:59.224+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-12T22:31:59.236+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: crypto_data_collection.fetch_and_store_crypto_data scheduled__2023-10-04T04:41:00+00:00 [queued]>
[2025-02-12T22:31:59.243+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: crypto_data_collection.fetch_and_store_crypto_data scheduled__2023-10-04T04:41:00+00:00 [queued]>
[2025-02-12T22:31:59.244+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 2
[2025-02-12T22:31:59.283+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): fetch_and_store_crypto_data> on 2023-10-04 04:41:00+00:00
[2025-02-12T22:31:59.290+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=14525) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-02-12T22:31:59.291+0000] {standard_task_runner.py:72} INFO - Started process 14533 to run task
[2025-02-12T22:31:59.290+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'crypto_data_collection', 'fetch_and_store_crypto_data', 'scheduled__2023-10-04T04:41:00+00:00', '--job-id', '5151', '--raw', '--subdir', 'DAGS_FOLDER/crypto_dag.py', '--cfg-path', '/tmp/tmpe8fe4f7x']
[2025-02-12T22:31:59.291+0000] {standard_task_runner.py:105} INFO - Job 5151: Subtask fetch_and_store_crypto_data
[2025-02-12T22:31:59.327+0000] {task_command.py:467} INFO - Running <TaskInstance: crypto_data_collection.fetch_and_store_crypto_data scheduled__2023-10-04T04:41:00+00:00 [running]> on host be7ba0dd3a47
[2025-02-12T22:31:59.395+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='crypto_data_collection' AIRFLOW_CTX_TASK_ID='fetch_and_store_crypto_data' AIRFLOW_CTX_EXECUTION_DATE='2023-10-04T04:41:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-10-04T04:41:00+00:00'
[2025-02-12T22:31:59.396+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-02-12T22:32:01.127+0000] {logging_mixin.py:190} INFO - Données récupérées : {'Response': 'Error', 'Message': 'You are over your rate limit please upgrade your account!', 'HasWarning': False, 'Type': 99, 'RateLimit': {'calls_made': {'second': 6, 'minute': 6, 'hour': 3737, 'day': 4631, 'month': 4631, 'total_calls': 4631}, 'max_calls': {'second': 20, 'minute': 300, 'hour': 3000, 'day': 7500, 'month': 11000}}, 'Data': {}}
[2025-02-12T22:32:01.130+0000] {logging_mixin.py:190} INFO - Erreur lors de l'insertion dans PostgreSQL : string indices must be integers, not 'str'
[2025-02-12T22:32:01.239+0000] {logging_mixin.py:190} INFO - Données uploadées dans MinIO.
[2025-02-12T22:32:01.240+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-02-12T22:32:01.246+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-02-12T22:32:01.247+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=crypto_data_collection, task_id=fetch_and_store_crypto_data, run_id=scheduled__2023-10-04T04:41:00+00:00, execution_date=20231004T044100, start_date=20250212T223159, end_date=20250212T223201
[2025-02-12T22:32:01.314+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-12T22:32:01.342+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-02-12T22:32:01.346+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
